content/docs/machine-learning/quickstart.md

```diff
@@ -0,0 +1,69 @@
+---
+sidebar_position: 2
+title: Get Started with Kubeflow
+description: Learn how to get started with Civo's Kubeflow via an example.
+---
+
+<head>
+  <title>Kubeflow quickstart | Civo Documentation</title>
+</head>
+
+In this quickstart mainly focused towards Pipelines and Notebooks, we take a look at a simple example to train and run a model using Kubeflow Pipelines and Notebooks.
```

Comment:
I would drop "simple" (it isn't simple) and not repeat the phrase "Pipelines and Notebooks":

In this quickstart, focused mainly on Pipelines and Notebooks, we take a look at an example to train and run a machine learning model using Kubeflow